src/DependencyInjection/Configuration.php

```diff
@@ -74,6 +74,7 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->scalarNode('serializer_adapter')->end()
```

Comment:
This approach would also simplify configuration, by making the serializer entirely optional.